files/build_templates/qos_config.j2

```diff
@@ -276,7 +296,12 @@
 {% endfor %}
 {% for port in PORT_ACTIVE %}
         "{{ port }}|6": {
+        {% if port in port_names_list_extra_queues %}
+            "scheduler"   : "scheduler.1",
+                    "wred_profile": "AZURE_LOSSLESS"
```

Comment:
fix the alignment

Reply:
The extra leading space here is to ensure the generated json has a good alignment. Not sure why the generated json has a different alignment with the template